ci-operator/step-registry/telcov10n/jiras/telcov10n-jiras-commands.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "************ telcov10n Fix user IDs in a container ************"
+[ -e "${HOME}/fix_uid.sh" ] && "${HOME}/fix_uid.sh" || echo "${HOME}/fix_uid.sh was not found" >&2
+
+function gathering_jiras_info {
+
+  echo "************ telcov10n Gathering Jiras info ************"
+
+  echo
+  echo "Jiras to watch out"
+  echo
+
+  jq -c '.[]' <<< "$(yq -o json <<< ${JIRAS})" | while read -r entry; do
+    # Extract the filename and content
+    description=$(echo "$entry" | jq -r '.description')
+    links=$(echo "$entry" | jq -r '.links')
+    pull_requets=$(echo "$entry" | jq -r '.PRs')
+
+     process_jiras "${description}" "${links}" "${pull_requets}"
+  done
+}
+
+function process_jiras {
+
+  desc=$1 ; shift
+  jiras=$1 ; shift
+  prs=$1
+
+  closed_jiras "${desc}" "${jiras}" "${prs}"
+
+  echo "####################################################################################################"
+  echo " -- Description ------------------------------------------------------------------------------------"
+  echo
+  echo "${desc}"
+  echo
+  echo " -- JIRAS ------------------------------------------------------------------------------------------"
+  echo
+  echo "${jiras}" | jq -r '.[]'
+  echo
+  echo " -- Related Pull Requests --------------------------------------------------------------------------"
+  echo
+  echo "${prs}" | jq -r '.[]'
+  echo
+  echo "####################################################################################################"
+  echo
+
+}
+
+function closed_jiras {
+
+  # TODO: Send a Slack notification or email to signal that a PreGA workaround
+  # should be reviewed
```

Comment:
@natifridman ,

I have tried to implement something very naive but good enough to track PreGA missing components. If any of you have an idea to extend it so that a notification can be sent in the even of a tracked Jira is closed, that would be of great help. WDYT?